config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = () => {
```

Comment:
Currently we still generate the `'use strict';` in these files, so please add it as well, including a empty line between that and the `module.exports` line.